### ISSUE TITLE
Use [ci build] rather than [ci skip]

### ISSUE THREAD

**Issue by RBWare** [travis-ci/travis-ci#5955]
I want to keep my projects from building unless I specifically ask it to. With `[ci skip]`, Travis knows not to build. I want it to not build, period, unless I specifically add `[ci build]` in the same way.

**Comment by BanzaiMan** [comment]
I think changing the default behavior is not a good idea. Even when it is a switch, it adds additional cognitive burden on both users and support.

**Comment by RBWare** [comment]
It doesn't have to be "default" behavior. It can be optional in the `.travis.yml` file. A setting that does that for your projects. The issue that I'm facing is that we have dozens of projects, that are all being constantly built, and it would be easier to just say, "only build when I tell you to", rather than building every time. @BanzaiMan

**Comment by BanzaiMan** [comment]
I meant "default behavior for the repo in question". Even if it is a setting per repo, it is a cognitive burden, not to mention the implementation cost.

If you find yourself overwhelmed, I suggest disabling building altogether, then triggering builds via API (https://docs.travis-ci.com/user/triggering-builds), which will trigger even when it is disabled.

**Comment by RBWare** [comment]
For our team, it's a cognitive burden to have to add `[ci skip]` each time we submit a change to a branch

**Comment by BanzaiMan** [comment]
@RBWare I suggest disabling builds, and using API to trigger builds.

**Comment by solarce** [comment]
@RBWare I am really curious about the workflow you're using that's led you to request this feature. 

Would you mind describing how your development workflow is implemented and share some details on why it's not desirable to have most of the commits to a branch trigger a build?

**Comment by RBWare** [comment]
Absolutely. I'll try to give as much detail as I can.

We have 2 travis instances available at any given time. We have approximately 20 projects being worked on by different teams. Each time a branch is pushed to github, a Travis build is triggered. One of the biggest issues we have been facing recently is that we're changing some of our iOS projects to use carthage. With carthage, build times for our iOS apps have spiked to ~40min/build. With only 2 travis instances available, and teams constantly submitting changes, our queue gets pretty lengthy pretty quickly. Another issue we're running into (that brought about this as a potential solution), is that we recently integrated with Fabric.io to push test / dev builds for our QA team. Because travis is dealing with so many builds, we're also flooding Fabric.io with a ton of different builds, sometimes builds that have very small changes. We recently communicated to our teams to start using `[ci skip]` in their commit messages, but there are a lot of times this is falling through the cracks. Our QA team tests and reviews features and changes prior to them being merged back into a base branch, so we need to have a way to setup builds for them, but they do not need dozens of them as changes get pushed between multiple dev for a single feature.

Hopefully that made sense. But it seemed like the easiest solution for us, is to disable the builds by default, but then when a team member specifies that they *want* a build after a commit, one is triggered, which would solve several of our issues at once. cc @solarce
